src/ripple/rpc/impl/ServerHandlerImp.h

```diff
@@ -169,11 +169,21 @@ class ServerHandlerImp
 
 private:
     bool
-    isWebsocketUpgrade (http_request_type const& request);
+    isWebsocketUpgrade(http_request_type const& request) const;
 
     bool
-    authorized (Port const& port,
-        std::map<std::string, std::string> const& h);
+    isStatusRequest(http_request_type const& request) const;
+
+    // Used with load balancers.
```

Comment:
Need a better description that this. Also, we usually put documentation for private members at the definition rather than the declaration.